crates/swift-bridge-ir/src/codegen/generate_rust_tokens/shared_enum.rs

```diff
@@ -130,14 +130,31 @@ impl SwiftBridgeModule {
             convert_ffi_variants_to_rust.push(convert_ffi_variant_to_rust);
         }
 
-        // TODO:
-        //  Parse any derives that the user has specified and combine those with our auto derives.
-        let automatic_derives = if shared_enum.has_one_or_more_variants_with_data() {
+        // Auto derives
+        let mut derives = if shared_enum.has_one_or_more_variants_with_data() {
             vec![]
         } else {
             vec![quote! {Copy}, quote! {Clone}]
```

Comment:
Hmm... I wish we didn't automatically derive anything and instead did it explicitly.. Especially given by comment below about restricting this to `Copy` enums for now.. 

We can worry about this later though.. Just sharing a thought. Does not need to impact this PR.